base/some.jl

```diff
@@ -138,8 +138,7 @@ true
 macro something(args...)
     expr = :(nothing)
     for arg in reverse(args)
-        expr = :((val = $arg) !== nothing ? val : $expr)
+        expr = :(val = $arg; $(GlobalRef(Base, :isnothing))(val) ? $expr : val)
```

Comment:
Did the change to `isnothing` help with inferrability? Is there a reason you explicitly want to do `Base.isnothing` instead of `isnothing`?

Adding parentheses mainly for clarity. Has no impact on the resulting expression though.
```suggestion
        expr = :(val = $arg; Base.isnothing(val) ? ($expr) : val)
```

Reply:
Seems like someone got escaping wrong: the `esc()` should go around the `$arg` inputs, not the `quote end` outputs

Reply:
> Did the change to `isnothing` help with inferrability? Is there a reason you explicitly want to do `Base.isnothing` instead of `isnothing`?

There is no inferrability different between them. I used `isnothing` because it offers possibility for users to extend this macro's functionality, but now I think there is no value for that since `something` is just defined for `::Nothing`.



> Seems like someone got escaping wrong: the `esc()` should go around the `$arg` inputs, not the `quote end` outputs

Yeah, addressed.